class.jetpack.php

```diff
@@ -2806,8 +2817,8 @@ public static function plugin_activation( $network_wide ) {
 	public static function do_version_bump( $version, $old_version ) {
 
 		if ( ! $old_version ) { // For new sites
-			// Setting up jetpack manage
-			Jetpack::activate_manage();
+			// There used to be stuff here, but this function might be useful to someone
```

Comment:
¯\_(ツ)_/¯ I left the conditional there to make it clear in the future how to add things for new sites only.  

I'm happy to add a deprecated call in here, but it will just need to be removed if this is used again.